test/Compiler.test.js

```diff
@@ -298,14 +298,14 @@ describe("Compiler", () => {
 				}
 			});
 		});
-		describe("static method", () => {
-			it("should have an accessible static method, Watching", (done) => {
-				const actual = Compiler.Watching(compiler, 1000, err => err);
-				actual.running.should.be.exactly(true);
-				actual.constructor.name.should.be.exactly("Watching");
-				done();
-			});
-		});
+		// describe("static method", () => {
+		// 	it("should have an accessible static method, Watching", (done) => {
+		// 		const actual = Compiler.Watching(compiler, 1000, err => err);
```

Comment:
Thanks for the review again. Also wondering, would you prefer if I squashed all the commits or just leave it as it is?